### DISCUSSION TITLE
Attributes not working in latest beta

### DISCUSSION THREAD

**Discussion by jasonpfrye** [livewire/livewire#6447] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.10

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Dispatch an event on a wire click
Create a listener for the event using the Attributes syntax
In beta9 the listener method fires, but in beta10 it will not. 

```
#[On('intro-questions')]
public function introFinished($status)
{
    $this->introQuestionsFinished = $status;
}
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by perisicnikola37** [comment]
Can you provide me the URL to your GitHub repository? 
And please describe a little bit more about the problem you are issuing
